packages/neo4j-driver-lite/src/index.ts

```diff
@@ -199,6 +202,21 @@ const {
  *         logger: (level, message) => console.log(level + ' ' + message)
  *       },
  *
+ *       // Configure a BookmarkManager for in the driver
+ *       //
+ *       // BookmarkManager is a piece of software responsible for keeping casual concistency between different sessions by sharing bookmarks
+ *       // between the them.
+ *       // Enabling it is done by supplying an BookmarkManager implementation instance to this param.
+ *       // A default implementation could be acquired by calling bookmarkManager factory function.
+ *       //
+ *       // **Warning**: Enabling Bookmark Manager can have a negative impact performance wise since all the queries will wait for the last changes
```

Comment:
```suggestion
 *       // **Warning**: Enabling the BookmarkManager can have a negative impact on performance since all the queries will wait for the latest changes
```